charts/super-agent-deployment/templates/rbac.yaml

```
@@ -5,7 +5,7 @@ kind: ClusterRole
 metadata:
   labels:
     {{- include "newrelic.common.labels" . | nindent 4 }}
-  name: {{ include "newrelic.common.naming.fullname" . }}
+  name: {{ include "newrelic.common.naming.fullname" . }}-flux
```

There is a [function in the common library to add suffixes](https://github.com/newrelic/helm-charts/blob/master/library/common-library/DEVELOPERS.md#newreliccommonnamingtruncatetodnswithsuffix)
```suggestion
  name: {{ include "newrelic.common.naming.truncateToDNSWithSuffix" (dict "name" (include "newrelic.common.naming.fullname" .) "suffix" "flux") }}
```